azure-pipelines.yml

```diff
@@ -0,0 +1,85 @@
+# Python package
+# Create and test a Python package on multiple Python versions.
+# Add steps that analyze code, save the dist with the build record, publish to a PyPI-compatible index, and more:
+# https://docs.microsoft.com/azure/devops/pipelines/languages/python
+
+trigger:
+  tags:
+    include:
+      - '*'
+  branches:
+    include:
+      - master
+      - release/*
+      - refs/tags/*
+pr:
+  - master
+  - release/*
+
+jobs:
+  - job: pytest
+    # how long to run the job before automatically cancelling
+    timeoutInMinutes: 45
+    # how much time to give 'run always even if cancelled tasks' before stopping them
+    cancelTimeoutInMinutes: 2
+
+    pool: gridai-spot-pool
+
+    # ToDo: this need to have installed docker in the base image...
+    #container: "pytorchlightning/pytorch_lightning:base-cuda-py$[ variables['python.version'] ]-torch1.6"
+    container:
+      image: "pytorchlightning/pytorch_lightning:base-cuda-py3.7-torch1.6"
+      #endpoint: azureContainerRegistryConnection
+      options: "--runtime=nvidia -e NVIDIA_VISIBLE_DEVICES=all"
+
+    workspace:
+      clean: all
+
+    steps:
+
+    - bash: |
+        lspci | egrep 'VGA|3D'
```

Comment:
`pciutils` needs to be installed to use `lspci`, otherwise we get:
```
/__w/_temp/6d67fc76-4b79-4645-ac4b-25a5145aeb08.sh: line 1: lspci: command not found
```
https://dev.azure.com/PytorchLightning/pytorch-lightning/_build/results?buildId=8250&view=logs&j=3afc50db-e620-5b81-6016-870a6976ad29&t=b8e11f49-5cfc-56d8-ba3e-ec2e1e8ddf1d&l=12

Reply:
Should we
1. `apt install pciutils` before this and use `lspci` to check devices, or
2. remove this line completely
?

Reply:
seems to be fine, this is included in the used docker container...